### PR TITLE
frontend: migrate build-chroots to flask-restx

### LINKED ISSUE #3000: Migrate api_v3/build-chroots to flask-restx
Migrate api_v3/build-chroots endpoint to flask-restx, included documentation, models and parsers.

### PR DESCRIPTION
**Pull request by nikromen**
Fixes #3000

### CODE REVIEW

File: frontend/coprs_frontend/coprs/views/apiv3_ns/schema/fields.py [hunk 1]
```diff
@@ -18,6 +18,13 @@
 # TODO: this file is not perfect in documenting... missing enums, choices, etc.
 
 
+# Feel free to drop this if you want to spend 4 hours why the fuck everything
+# stopped working. Long story short - query_to_params and flask_restx.fields.Url.output
+# aren't friends.
+class Url(String):
```
Comment (nikromen) [inline frontend/coprs_frontend/coprs/views/apiv3_ns/schema/fields.py:24]: almost got mental breakdown from this one :D explanation TODO

File: frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_build_chroots.py [hunk 2]
```diff
@@ -36,41 +55,89 @@ def build_config(build_chroot):
...
+        """
+        Get build chroot
+        Get information about specific build chroot by build id and mock chroot name.
+        """
+        chroot = ComplexLogic.get_build_chroot(build_id, chrootname)
+        return to_dict(chroot)
+
+
+@apiv3_bchroots_ns.route("/list")
+@apiv3_bchroots_ns.route("/list/<int:build_id>")  # TODO: ask if this is also deprecated?
```
Comment (nikromen) [inline frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_build_chroots.py:81]: I saw deprecated comments to these routes but not in this one - should I also mark it as deprecated in documentation?

Comment (FrostyX) [reply frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_build_chroots.py:81]: I think it would make sense to mark this deprecated as well.

**Review by FrostyX — COMMENTED** [review]
Otherwise LGTM

File: frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_build_chroots.py (hunk 2 above)
Comment (nikromen) [reply frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_build_chroots.py]: done, PTAL

**Review by FrostyX — APPROVED** [review]

**Pull request merged**